Graphs/Planar Graphs/Graph Embeddings/eulers_genus.concept.yaml

```
@@ -0,0 +1,30 @@
+ru: Род Эйлера
+en: Euler genus
+
+definition:
+  ru:
+    Характеристика поверхности, определяемая как число вершин минус число ребер
+    плюс число граней
+  en:
+    A characteristic of a surface defined as the number of vertices minus the
+    number of edges plus the number of faces
+  using:
+    concepts: |
+      surface
+      graph
+      vertex
+      edge
+    nrels: |
+      face
+
+statement:
+  euler_theorem:
+    title:
+      ru: Теорема Эйлера
+      en: Eulers Theorem
```

```suggestion
  planarity_genus_theorem:
    title:
      ru: Теорема о планарности и роде
      en: Planarity-Genus Theorem
```
The statement "If a graph is planar, then its genus is equal to one" is not Euler's Theorem, but rather a special case of the more general Euler's formula for planar graphs.